app/assets/stylesheets/application.css

```diff
@@ -24,4 +24,9 @@
  *= require comments
  *= require wiki
  *= require question
+ *= require dashboard
+ *= require fancy
```

Comment:
Hi, some of these are actually connected into logic to enable/disable them based on context; take a look at these lines:

https://github.com/publiclab/plots2/blob/eec1b54f9b78f13bd12fa11466f5b2823f7dfed0/app/views/layouts/application.html.erb#L25-L28

The others I think are included in directly from sub-templates based on context, so we don't use them too often or serve them to every page on the site. This isn't perfect, but it's attempting to keep some CSS use-specific. 

Because of this, I'm not sure this PR is needed? I appreciate the idea though! If you have other suggestions for how to make the CSS cleaner, more readable or maintainable, I'm very open to it -- maybe open an issue for this -- because over the years it's become a bit less manageable. 👍